tailormap-components/projects/core/src/lib/user-interface/edit-bar/add-feature-menu/add-feature-menu.component.css

```diff
@@ -1,9 +1,21 @@
 .container {
-  display: flex; /* or inline-flex */
+  display: flex;
   flex-direction: row;
+  align-items: center;
 }
 
-.container button {
-  margin-left: 5px;
-  margin-right: 5px;
+tailormap-menu-button,
```

Comment:
https://sonarcloud.io/project/issues?id=B3Partners_tailormap&open=AXfbSiOMS8UujDGouequ&pullRequest=2425&resolved=false&types=BUG

Reply:
correct me if I'm wrong, maar `tailormap-menu-button` is een custom html component, dus deze selector is correct. Zie bijv. https://github.com/B3Partners/tailormap/blob/87919ddccb87bb93e0d2d691ca1220c14f23bf58/tailormap-components/projects/core/src/lib/user-interface/edit-bar/add-feature-menu/add-feature-menu.component.html#L12-L16